dev/com.ibm.ws.jbatch.cdi.2.1/src/com/ibm/ws/jbatch/cdi/BatchProducerBean.java

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright 2013, 2022 International Business Machines Corp.
```

Comment:
```suggestion
 * Copyright 2013, 2023 International Business Machines Corp.
```
Should this be EPL 2.0 license?